raft/raftpb/raft.proto

```diff
@@ -76,13 +76,15 @@ message HardState {
 }
 
 message ConfState {
-	repeated uint64 nodes = 1;
+	repeated uint64 nodes    = 1;  // Voters
```

Comment:
remove this comment. `voter` is meaningless without any context. or we need to explain exactly what is nodes.